src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -690,14 +690,8 @@ namespace ZoneAirLoopEquipmentManager {
                 if (InNodeNum > 0) {
                     MassFlowRateMaxAvail = state.dataLoopNodes->Node(InNodeNum).MassFlowRateMaxAvail;
                     MassFlowRateMinAvail = state.dataLoopNodes->Node(InNodeNum).MassFlowRateMinAvail;
-                    AirLoopNum = state.dataDefineEquipment->AirDistUnit(AirDistUnitNum).AirLoopNum;
-                    if (AirLoopNum > 0) {
-                        DesFlowRatio = state.dataAirLoop->AirLoopFlow(AirLoopNum).SysToZoneDesFlowRatio;
-                    } else {
-                        DesFlowRatio = 1.0;
-                    }
                     MassFlowRateUpStreamLeakMax = max(state.dataDefineEquipment->AirDistUnit(AirDistUnitNum).UpStreamLeakFrac *
-                                                          state.dataLoopNodes->Node(InNodeNum).MassFlowRateMax * DesFlowRatio,
+                                                          state.dataLoopNodes->Node(InNodeNum).MassFlowRateMaxAvail,
                                                       0.0);
```

Comment:
The upstream leakage rate is directly calculated from the maximum available air flow rate. No need for local variable `DesFlowRatio`.

Reply:
We did talk about this and I agreed with you at the time. But as I think about this more the leakage in a duct would be driven by duct static pressure and E+ assumes that the duct static pressure remains constant. This method assumes duct static pressure varies with air flow rate. If I blocked off one zone diffuser in a VAV system would there be leakage in that section of duct (develop says yes) or not (this method says no)?

Reply:
It may also be that a VAV system's leakage response is what develop does now. And a constant volume system using a cycling fan would respond as this branch suggests.

Reply:
You brought up interesting points: 
- If a static pressure rest is considered, the leakage amount is constant, not the leakage fraction, because the leakage amount is driven by the pressure difference, which is constant with static reset.
- otherwise, the leakage amount may vary with the duct pressure, which is dependent on the supply air flow rate

Hmmm, the documentation is also consistent with the current code.